app/code/Magento/Backup/Model/ResourceModel/Table/GetListTables.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Backup\Model\ResourceModel\Table;
+
+use Magento\Framework\App\ResourceConnection;
+
+/**
+ * Class GetListTables
+ */
+class GetListTables
+{
+    const TABLE_TYPE = 'BASE TABLE';
```

Comment:
use private constants to prevent polluting public contract with excessive obligations

Reply:
Fixed